src/main/java/com/gotocompany/firehose/utils/KafkaProducerTypesMetadata.java

```diff
@@ -0,0 +1,17 @@
+package com.gotocompany.firehose.utils;
+
+import java.util.regex.Pattern;
+
+public enum KafkaProducerTypesMetadata {
+    DLQ("DLQ_KAFKA_+");
```

Comment:
`_+` not sure why you've changed from `DLQ_KAFKA_` to this? 

Final pattern would be `^DLQ_KAFKA_+(.*)` how this plus would have impact here for multiple/consequent underscores? This is bit ambiguous.

Reply:
Oh this one to cater multiple `_`. Suppose there is `DLQ_KAFKA___MAX_POLL_RECORDS` it will also be parsed. Should I revert to the existing one?

Reply:
That is fine, but what is the significance of + here? is it a special character in RegEx?

Reply:
okay it's a special character in RegEx. It signifies `one or more preceding char`. 

Suppose we have this regex : `DLQ_KAFKA_+(.*)`
It will be able to capture : `DLQ_KAFKA_____SOME_VALID_PROPS`